.github/ISSUE_TEMPLATE/Bug_report.md

```diff
@@ -20,16 +20,13 @@ A clear and concise description of what you expected to happen.
 **Screenshots**
 If applicable, add screenshots to help explain your problem.
 
-**Desktop/Laptop (please complete the following information):**
- - OS: [e.g. MacOS 10.13.3]
- - Browser: [e.g. chrome, safari]
- - Version: [e.g. 22]
-
-**Smartphone (please complete the following information):**
- - Device: [e.g. iPhone6]
- - OS: [e.g. iOS8.1]
- - Browser: [e.g. stock browser, safari]
- - Version: [e.g. 22]
+**Environment information**
+ - AUT version: [e.g. 0.16.0, HEAD]
+ - OS: [e.g. MacOS 10.13.3, Ubuntu 18.04]
+ - Java version: [e.g. Java 8]
+ - Apache Spark version: [e.g 2.1.3, 2.3.1]

```

Comment:
I think the changes to the template work really well @ruebot. Two tiny tweaks to offer:
e.g  --> e.g.